src/registrar/templates/includes/ao_example__exec.html

```diff
@@ -0,0 +1,3 @@
+<p>Domain requests from executive branch agencies must be authorized by <strong>CIOs</strong> or <strong>agency heads</strong>.</p>
```

Comment:
I would very much prefer that all abbreviations be [eliminated or at least defined on first usage](https://www.plainlanguage.gov/guidelines/words/minimize-abbreviations/) (within a page):

`<strong>Chief Information Officers</strong>` or 

`<strong>Chief Information Officers (<abbr>CIO</abbr>s)</strong>`

CC @michelle-rago